lib/person.rb

```diff
@@ -0,0 +1,43 @@
+require 'yaml'
+require 'date'
+require 'pry'
+require './lib/library.rb'
+
+class Person
+
+attr_accessor :books
+
+def initialize
+  @books = YAML.load_file('./lib/person.yml')
+end
+
+def perform_returnbook(title)
+  @books.detect do |x|
+    if x[:item][:title] == title
+      index = @books.index(x)
+      @books.delete_at(index)
+      # File.open('./lib/person.yml', 'w') { |f| f.write @books.to_yaml }
+    end
+  end
+end
+
+def watch_list(library)
+  library.see_list
+end
+
+def perform_borrow(title)
+  book_borrow(title, available: true, return_date: Date.today + 30)
+  # File.open('./lib/person.yml', 'w') { |f| f.write @books.to_yaml }
+end
```

Comment:
Same thing as in the other code - you should not have a method whose only job is to call another method.